### PR TITLE
fix(codewhisperer): Fixes some tests that fail when run locally

### PR DESCRIPTION
**Pull request by nkomonen-amazon**
When unit tests are run locally + CW is setup with Builder ID the `credentialStartUrl` value will be defined, this causes unit tests to fail.

To fix it, this change adds to the assertion whatever the current `startUrl` is.

- Additionally, some tests assume that CW connection is not configured automatically. This ensures the connection is not set up and prevents any failures due to the test env.

For more info see sim IDE-10578

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by justinmk3 — APPROVED** [review]
waiting for CW approval

File: src/test/codewhisperer/explorer/codewhispererNode.test.ts
Comment (nkomonen-amazon) [inline src/test/codewhisperer/explorer/codewhispererNode.test.ts:17]: This block of code sets lots of auth stuff to not being connected initially, any following test that needs it to be connected (returns true) will set it on its own.

**Review by leigaol — APPROVED** [review]

**Pull request merged**
